so_vector/files.txt

```diff
@@ -0,0 +1,2 @@
+documents.json
```

Comment:
Do we need this file?

Reply:
Maybe not, I just followed the example from several other tracks (pmc, so, tsdb, etc.)

Reply:
Oh I see, I have never used this script before: https://github.com/elastic/rally-tracks/commit/b5e11e2e3890b5827575b4551ce23b1b20f32c81